README.md

```diff
@@ -171,25 +173,23 @@ want.
   ]
 }
 ```
+<br />
 
-</details>
-</details>
+#### Text dataset format
 
-<details>
-<summary>Text</summary>
-<br />
-<details>
-<summary>.csv, .xls/.xlsx</summary>
+- dataset must have an **ID column** (`review_id` in the example below) - a column containing identifiers that uniquely identify each row.
+- dataset must have a **text column** (`review` in the example below) that serves as the input set for the prediction task.
```

Comment:
```suggestion
- dataset must have a **text column** (`review` in the example below) that serves as the sole predictive feature for modeling the label and identifying label issues.
```

Reply:
Clarify that:

Beyond the **text column**, the dataset can contain other columns with extra miscellaneous metadata that will be ignored when modeling the labels.